libclamav_rust/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# preserve signature for .cargo/vendor files
+* -text
```

Comment:
The desire for this change makes sense to me, to prevent git from doing end-of-line conversions for any files found in the `.cargo/vendor` directory. 

I'm not sure that adding a file here, or having it apply to all files under `libclamav_rust` makes sense. 

Instead, i wonder if the patch should be:
```diff
diff --git a/.gitattributes b/.gitattributes
index 2253303d2..1cf4e5d9c 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -7,7 +7,8 @@
 #

 # Files that should be left untouched (binary is macro for -text -diff)
-*.ref           binary
+*.ref                           binary
+libclamav_rust/.cargo/vendor    binary

 #
 # Exclude files from exporting
 ```
 
 or maybe just
 ```diff
diff --git a/.gitattributes b/.gitattributes
index 2253303d2..555b0ce6e 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -8,6 +8,7 @@

 # Files that should be left untouched (binary is macro for -text -diff)
 *.ref           binary
+.cargo/vendor   binary

 #
 # Exclude files from exporting
```

Reply:
Ideally, whatever change we make here, we need .gitattributes not effectively alter files shipped in the tarball. Even better would be for .gitattributes to NOT ship in the tarball. Is that possible?

Reply:
Indeed removing the `.gitattributes` from the tarball would also solve the issue as I noted in the commit message.
but your suggestion is right, using a single `.gitattributes` file is easier for maintenance and could benefits any other locations where `cargo` packages files are added to the tarball

Reply:
To be honest, I misunderstood about removing the `.gitattributes` from the tarball.  But I am happy enough with this solution to explicitly leave the `.cargo/vendor` directory alone. 

Thank you for submitting this pr!